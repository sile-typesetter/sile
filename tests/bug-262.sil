\begin[papersize=a5]{document}
\nofolios
\script[src=packages/lorem]
\hbox
\begin[first-content-frame=a]{pagetemplate}
  \frame[id=a,left=2%pw,right=44%pw,top=2%ph,bottom=98%ph,next=b]
  \frame[id=b,left=46%pw,right=98%pw,top=2%ph,bottom=98%ph]
\end{pagetemplate}
\skip[height=43em] % change this to 45 for no bug symptoms
\lorem[words=80]\par
{
\set[parameter=document.lskip,value=4em]
This is an indented environment
}
\end{document}
