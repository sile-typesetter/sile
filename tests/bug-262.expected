Set paper size 	419.5275636	595.275597
Begin page
Mx 	8.3906
My 	11.9055
Mx 	28.3906
My 	454.9055
Set font 	Gentium Plus;10;400;;normal;;LTR
T	79 82 85 72 80	(lorem)
Mx 	56.5801
T	76 83 86 88 80	(ipsum)
Mx 	85.6192
T	71 82 79 82 85	(dolor)
Mx 	111.3869
T	86 76 87	(sit)
Mx 	125.2405
T	68 80 72 87	(amet)
Mx 	149.7630
T	70 82 81	(con)
Mx 	164.6800
T	86 72	(se)
Mx 	173.1615
T	87 72	(te)
Mx 	181.2230
T	16	(-)
Mx 	8.3906
My 	467.9055
T	87 88 85	(tur)
Mx 	24.4014
T	86 68 71 76 83 86 70 76 81 74	(sadipscing)
Mx 	70.7100
T	72 79 76 87 85	(elitr)
Mx 	91.4620
T	86 72 71	(sed)
Mx 	108.4543
T	71 76 68 80	(diam)
Mx 	132.2971
T	81 82 81 88 80 92	(nonumy)
Mx 	169.9339
T	72 76 85	(eir)
Mx 	181.2230
T	16	(-)
Mx 	8.3906
My 	480.9055
T	80 82 71	(mod)
Mx 	30.4683
T	87 72 80 83 82 85	(tempor)
Mx 	64.6651
T	76 81 89 76 71 88 81 87	(invidunt)
Mx 	103.8082
T	88 87	(ut)
Mx 	116.3645
T	79 68 69 82 85 72	(labore)
Mx 	146.1668
T	72 87	(et)
Mx 	158.0443
T	71 82 79 82 85 72	(dolore)
Mx 	8.3906
My 	493.9055
T	80 68 74 81 68	(magna)
Mx 	38.5192
T	68 79 76 84 88 92 68 80	(aliquyam)
Mx 	79.0531
T	72 85 68 87	(erat)
Mx 	98.1857
T	86 72 71	(sed)
Mx 	114.3885
T	71 76 68 80	(diam)
Mx 	137.4420
T	89 82 79 88 83 87 88 68	(voluptua)
Mx 	176.5599
T	68 87	(at)
Mx 	8.3906
My 	506.9055
T	89 72 85 82	(vero)
Mx 	29.2677
T	72 82 86	(eos)
Mx 	45.1155
T	72 87	(et)
Mx 	55.5141
T	68 70 70 88 86 68 80	(accusam)
Mx 	92.9635
T	72 87	(et)
Mx 	103.3621
T	77 88 86 87 82	(justo)
Mx 	125.9189
T	71 88 82	(duo)
Mx 	143.7834
T	71 82 79 82 85 72 86	(dolores)
Mx 	176.5306
T	72 87	(et)
Mx 	8.3906
My 	519.9055
T	72 68	(ea)
Mx 	20.7225
T	85 72 69 88 80	(rebum)
Mx 	50.8328
T	86 87 72 87	(stet)
Mx 	69.3219
T	70 79 76 87 68	(clita)
Mx 	90.2672
T	78 68 86 71	(kasd)
Mx 	111.9204
T	74 88 69 72 85 74 85 72 81	(gubergren)
Mx 	157.8510
T	81 82	(no)
Mx 	171.5208
T	86 72 68	(sea)
Mx 	8.3906
My 	532.9055
T	87 68 78 76 80 68 87 68	(takimata)
Mx 	47.2594
T	86 68 81 70 87 88 86	(sanctus)
Mx 	80.7963
T	72 86 87	(est)
Mx 	95.3146
T	79 82 85 72 80	(lorem)
Mx 	122.2596
T	76 83 86 88 80	(ipsum)
Mx 	150.0543
T	71 82 79 82 85	(dolor)
Mx 	174.5775
T	86 76 87	(sit)
Mx 	8.3906
My 	545.9055
T	68 80 72 87	(amet)
Mx 	32.1593
T	79 82 85 72 80	(lorem)
Mx 	59.5951
T	76 83 86 88 80	(ipsum)
Mx 	87.8805
T	71 82 79 82 85	(dolor)
Mx 	112.8944
T	86 76 87	(sit)
Mx 	125.9942
T	68 80 72 87	(amet)
Mx 	149.7630
T	70 82 81	(con)
Mx 	164.6800
T	86 72	(se)
Mx 	173.1615
T	87 72	(te)
Mx 	181.2230
T	16	(-)
Mx 	8.3906
My 	558.9055
T	87 88 85	(tur)
Mx 	24.4014
T	86 68 71 76 83 86 70 76 81 74	(sadipscing)
Mx 	70.7100
T	72 79 76 87 85	(elitr)
Mx 	91.4620
T	86 72 71	(sed)
Mx 	108.4543
T	71 76 68 80	(diam)
Mx 	132.2971
T	81 82 81 88 80 92	(nonumy)
Mx 	169.9339
T	72 76 85	(eir)
Mx 	181.2230
T	16	(-)
Mx 	8.3906
My 	571.9055
T	80 82 71	(mod)
Mx 	30.4683
T	87 72 80 83 82 85	(tempor)
Mx 	64.6651
T	76 81 89 76 71 88 81 87	(invidunt)
Mx 	103.8082
T	88 87	(ut)
Mx 	116.3645
T	79 68 69 82 85 72	(labore)
Mx 	146.1668
T	72 87	(et)
Mx 	158.0443
T	71 82 79 82 85 72	(dolore)
Mx 	192.9827
My 	19.4739
T	80 68 74 81 68	(magna)
Mx 	223.7188
T	68 79 76 84 88 92 68 80	(aliquyam)
Mx 	264.8601
T	72 85 68 87	(erat)
Mx 	284.6001
T	86 72 71	(sed)
Mx 	301.4104
T	71 76 68 80	(diam)
Mx 	325.0713
T	89 82 79 88 83 87 88 68	(voluptua)
Mx 	364.7967
T	68 87	(at)
Mx 	375.9576
T	89 72 85 82	(vero)
Mx 	397.6263
T	72 82 86	(eos)
Mx 	192.9827
My 	31.4739
T	72 87	(et)
Mx 	203.6957
T	68 70 70 88 86 68 80	(accusam)
Mx 	241.4595
T	72 87	(et)
Mx 	252.9827
My 	313.3487
T	55 75 76 86	(This)
Mx 	273.2242
T	76 86	(is)
Mx 	282.4402
T	68 81	(an)
Mx 	295.1915
T	76 81 71 72 81 87 72 71	(indented)
Mx 	334.6615
T	72 81	(en)
Mx 	344.7982
T	89 76	(vi)
Mx 	352.4398
T	85 82 81	(ron)
Mx 	366.9466
T	80 72 81 87	(ment)
End page
Finish
